File: m/rls/strem.r
file	st_rem - remove leading substring
include rid:rider
include rid:stdef

;	res = st_rem (mod, tar)
;	
;	mod : * char
;		model string pointer
;	tar : * char
;		target string pointer
;
;	res : * char
;		at start of target if substring removed
;		<> if scan failed to match target

  func	st_rem
	mod : * char			; model
	tar : * char~			; target
	()  : * char			; result
  is	pst : * char~			;
	pst = st_scn (mod, tar)		; attempt to match it
	pass fail			; not present
	st_mov (pst, tar)		; delete it
	reply tar			; found it
  end


File: m/rls/rlb/strem.c
/* file -  st_rem - remove leading substring */
#include "m:\rid\rider.h"
#include "m:\rid\stdef.h"
char *st_rem(
register char *mod ,
register char *tar )
{ char *pst ;
  if( (pst = st_scn (mod, tar)) == NULL)return ( NULL );
  st_mov (pst, tar);
  return ( tar);
} 
